proc_scripts/baseline.py

```diff
@@ -1,8 +1,10 @@
+from pyspecdata import *
+logger=init_logging('info')
```

Comment:
as elsewhere -- `import logging` here, then `logging.info(`, etc, in the code.

this command would initiate logging, which should only be done in the main example script.